### DISCUSSION TITLE
can I skip files from test coverage

### DISCUSSION THREAD

**Discussion by AsmaaMagdHelali** [zgosalvez/github-actions-report-lcov#234] [General]
I want to skip coverage for some files, can I be able to do that

**Comment by sflanker** [comment]
I think you're best option is to configure the code coverage tool you are using to ignore these files and not include them in your lcov report file (ie https://github.com/istanbuljs/nyc?tab=readme-ov-file#common-configuration-options or https://github.com/bcoe/c8?tab=readme-ov-file#cli-options--configuration).
